scripts/variantstore/wdl/GvsSitesOnlyVCF.wdl

```diff
@@ -126,24 +142,54 @@ task SitesOnlyVcf {
     output {
         File output_vcf="~{output_filename}"
         File output_vcf_idx="~{output_vcf_idx}"
+        File annotations_template="~{custom_annotations_template}"
     }
 }
 
-task AnnotateShardedVCF {
+task ExtractAnAcAfFromVCF {
+    input {
+        File input_vcf
+        File input_vcf_index
+        File custom_annotations_template
+    }
+    String custom_annotations_file_name = "an_ac_af.tsv"
+    # separate multi-allelic sites into their own lines, remove deletions and extract the an/ac/af
+    command <<<
+        set -e
+        cp ~{custom_annotations_template} ~{custom_annotations_file_name}
+        bcftools norm -m- ~{input_vcf} | bcftools query -f '%CHROM\t%POS\t%REF\t%ALT\t%AN\t%AC\t%AF\n' | grep -v "*" >> ~{custom_annotations_file_name}
+    >>>
+    # ------------------------------------------------
+    # Runtime settings:
+    runtime {
+        docker: "biocontainers/bcftools:v1.9-1-deb_cv1" # TODO lets put bcftools in our custom container please
```

Comment:
If this is really something we want to do, please create a "tech debt" ticket for this and remove this comment.  I think it might be more likely to get done that way.

Reply:
good point!